packages/flutter/test/cupertino/colors_test.dart

```diff
@@ -167,7 +167,7 @@ void main() {
   });
 
   test('can resolve null color', () {
-    expect(CupertinoDynamicColor.resolve(null, null), isNull);
+    expect(CupertinoDynamicColor.resolve(null, _NullElement.instance), isNull);
```

Comment:
To prevent null to be used. The `CupertinoDynamicColor.resolve` user now know that the context is mandatory to call this function. Calling it with null resolvable looks quite uncommon and using a non-null value required the context to be non-null.

Reply:
I see, the second `CupertinoDynamicColor.resolve()` argument can no longer be null. But the first (Color) parameter can, so this test survives.

Reply:
Exactly :)